sys-fs/udiskie/udiskie-1.7.5.ebuild

```diff
@@ -23,11 +23,7 @@ DEPEND="app-text/asciidoc
 	dev-python/setuptools[${PYTHON_USEDEP}]"
 
 src_prepare() {
-	sed -i \
-		-e "s:udiskie.__version__:2:" \
-		-e 's:gtk-update-icon-cache:true:' \
-		setup.py || die
-
+	setup.py || die
```

Comment:
Doesn't this die for you?

Reply:
It does in fact. Had a problem with my setup - should be fixed now.